images/runtime/php-fpm/generateDockerfiles.sh

```diff
@@ -45,6 +45,6 @@ do
 	sed -i "s|$PHP_VERSION_PLACEHOLDER|$PHP_VERSION|g" "$TARGET_DOCKERFILE_BASE"
 
 	# RUNTIME_BASE_IMAGE_TAG="php-$VERSION_DIRECTORY-fpm-$PHP_FPM_RUNTIME_BASE_TAG-$ImageDebianFlavor"
-    RUNTIME_BASE_IMAGE_TAG="php-$VERSION_DIRECTORY-fpm-$PHP_FPM_RUNTIME_BASE_TAG"
+    RUNTIME_BASE_IMAGE_TAG="php-$VERSION_DIRECTORY-fpm-debian-$ImageDebianFlavor-$PHP_FPM_RUNTIME_BASE_TAG"
```

Comment:
Ah, good catch. The fact that this base image is changed was unintentional. None of the base images were intended to be changed to include the OS type. Unfortunately, php-fpm uses [a txt file](https://github.com/microsoft/Oryx/blob/main/images/runtime/php-fpm/8.1/tag.txt) to determine what the tag is going to be, and I did not account for the fact that this file would also change the base image tag. I believe that this is the only type that does this, so the other bases should continue to not have the os type in their tags. We may want to update all of them to be consistent, but for now this should work

Reply:
No worries 👍 I just filed a work item [here](https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1656098) to investigate if this is a problem for other platforms' runtime images.